release-notes: add 5.2.1 release notes (#6438)
pingcap/docs: releases/release-5.2.1.md

---
title: TiDB 5.2.1 Release Notes
---

# TiDB 5.2.1 Release Notes

> Release date: September x, 2021

Reviewer: ```suggestion
Release date: September 9, 2021
```

TiDB version: 5.2.1

> ## Bug Fixes

Reviewer: ```suggestion
## Bug fixes
```

+ TiDB

>     - Fix the error report of a wrong execution plan caused by the shallow copy of schema columns when pushing down the aggregation operators in partition tables [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)

Reviewer: ```suggestion
    - Fix an error that occurs during execution caused by the wrong execution plan. The wrong execution plan is caused by the shallow copy of schema columns when pushing down the aggregation operators on partitioned tables. [#27797](https://github.com/pingcap/tidb/issues/27797) [#26554](https://github.com/pingcap/tidb/issues/26554)
```

+ TiKV

>     - Fix the issue of unavailable TiKV caused by Raftstore deadlock when migrating Regions. You can deal with this issue by closing the scheduling or restarting the unavailable TiKV [#10909](https://github.com/tikv/tikv/issues/10909)

Reviewer: ```suggestion
    - Fix the issue of unavailable TiKV caused by Raftstore deadlock when migrating Regions. The workaround is to disable the scheduling and restart the unavailable TiKV. [#10909](https://github.com/tikv/tikv/issues/10909)
```
